packages/core/helper-plugin/lib/src/components/DateTimePicker/index.js

```diff
@@ -1,213 +1,17 @@
-import React, { useEffect, useState } from 'react';
-import PropTypes from 'prop-types';
-import styled from 'styled-components';
-import { Stack } from '@strapi/design-system/Stack';
-import { DatePicker } from '@strapi/design-system/DatePicker';
-import { TimePicker } from '@strapi/design-system/TimePicker';
-import { Field, FieldHint, FieldLabel, FieldError } from '@strapi/design-system/Field';
-import { Flex } from '@strapi/design-system/Flex';
-import { Box } from '@strapi/design-system/Box';
-import { parseDate } from './parseDate';
-
-const CustomField = styled(Field)`
-  > div {
-    > div {
-      p[data-strapi-field-error='true'] {
-        display: none;
-      }
-    }
-  }
-`;
-
-const LabelAction = styled(Box)`
-  svg path {
-    fill: ${({ theme }) => theme.colors.neutral500};
-  }
-`;
-
-const DateTimePicker = ({
-  ariaLabel,
-  clearLabel,
-  disabled,
-  error,
-  hint,
-  label,
-  labelAction,
-  onChange,
-  onClear,
-  name,
-  required,
-  size,
-  step,
-  value,
-  ...props
-}) => {
-  const initialDate = parseDate(value);
-  const [dateValue, setDateValue] = useState(initialDate);
-  const [timeValue, setTimeValue] = useState(
-    initialDate
-      ? `${initialDate.getHours()}:${initialDate.getMinutes()}:${initialDate.getSeconds()}`
-      : null
-  );
-
-  const handleDateChange = (e) => {
-    setDateValue(e);
-
-    if (timeValue) {
-      const dateToSet = new Date(e);
-      dateToSet.setHours(timeValue.split(':')[0]);
-      dateToSet.setMinutes(timeValue.split(':')[1]);
-
-      if (onChange) {
-        onChange(dateToSet);
-      }
-    } else {
-      const dateToSet = new Date(e);
-      setTimeValue(`${dateToSet.getHours()}:${dateToSet.getMinutes()}:${dateToSet.getSeconds()}`);
-
-      if (onChange) {
-        onChange(dateToSet);
-      }
-    }
-  };
-
-  const handleTimeChange = (e) => {
-    setTimeValue(e);
-
-    const dateToSet = new Date(dateValue);
-    dateToSet.setHours(e.split(':')[0]);
-    dateToSet.setMinutes(e.split(':')[1]);
-
-    if (!dateValue) {
-      setDateValue(new Date());
-    }
-
-    if (onChange) {
-      onChange(dateToSet);
-    }
-  };
-
-  const handleDateClear = () => {
-    setDateValue(undefined);
-    setTimeValue(undefined);
-    onClear();
-  };
-
-  const handleTimeClear = () => {
-    setTimeValue(undefined);
-
-    let dateToSet;
-
-    if (dateValue) {
-      dateToSet = new Date(dateValue);
-      dateToSet.setHours('00');
-      dateToSet.setMinutes('00');
-    }
-
-    if (onChange) {
-      onChange(dateToSet);
-    }
-  };
+import React, { useEffect } from 'react';
+import { DateTimePicker as MyDateTimePicker } from '@strapi/design-system';
 
+export default function DateTimePicker(props) {
```

Comment:
```suggestion
// TODO: remove DateTimePicker component from the helper-plugin in V5
export default function DateTimePicker(props) {
```

Instead, write it here.

Reply:
ok thanks, done